src/lib/openapi-to-json-schema.ts

```diff
@@ -310,6 +310,14 @@ export default function toJSONSchema(
         schema = mergeJSONSchemaAllOf(schema as RMOAS.JSONSchema, {
           ignoreAdditionalProperties: true,
           resolvers: {
+            // `merge-json-schema-allof` by default takes the first `description` when you're
+            // merging an `allOf` but because generally when you're merging two schemas together
+            // with an `allOf` you want data in the subsequent schemas to be applied to the first
+            // and `description` should be a part of that.
+            description: (obj: string[]) => {
+              return obj.slice(-1)[0];
+            },
```

Comment:
Here's the line in `json-schema-merge-allof` where it's always using the first description instead of the last. https://github.com/mokkabonna/json-schema-merge-allof/blob/master/src/index.js#L229

Reply:
It also does this same behavior for `$id`, `$ref`, `$schema`, `default`, and `title`. I think the first[^1] behavior is fine for `$id`, `$ref`, and `$schema`, but we might want to also do this same last pick for `default` and `title`.

[^1]: https://www.youtube.com/watch?v=ciG-Xs7mBwU